src/js/gameController.js

```diff
@@ -268,30 +262,12 @@ var gameController = {
         return  67200*(1/(1+Math.exp(-0.1*(time-30)))+1);
     },
 
-    setEggProbabilities: function(regularEggPr, bombPr, scoreBoostPr, frenzyPr, comboPr, oneUpPr){
-        this.regularEggProb = regularEggPr;
-        this.bombProb = bombPr;
-        this.scoreBoostProb = scoreBoostPr;
-        this.frenzyProb = frenzyPr;
-        this.comboProb = comboPr;
-        this.oneUpProb = oneUpPr;
-    },
-
-    calculateEggProbWithOrWithoutOneUp(){
-        if(this.lives<this.maxLives){
-            playState.calculateEggProbWithOneUP();
-        } else {
-            playState.calculateEggProbWithoutOneUP();
-        }
-    },
-
     /**
      * Resets all aspects of the game when the user starts over
      */
     resetGameComponents: function(){
-        this.currentTime=0;
+        gameController.currentGameStartTimeSecs = game.time.now / 1000; 
```

Comment:
Since currentGameStartTimeSecs is a variable in gameController.js let's try and use this for better consistency within the codebase